src/main/java/me/zeroeightsix/fiber/NodeOperations.java

```diff
@@ -8,8 +8,9 @@
 public class NodeOperations {
 
     /**
-     * Merges two {@link ConfigNode} objects
-     * @param from  The ConfigNode that will be read from, but not mutated.
+     * Merges two {@code ConfigNode} objects.
```

Comment:
Maybe add 'into a new object'? Since that seems to be what happens but not really sure.

Reply:
It actually mutates the second parameter, `to`.
I'll add more text explaining this behaviour.